src/screens/Analytics/GlobalSearchResults/Tables/PaymentIntent/PaymentIntentTable.res

```diff
@@ -0,0 +1,167 @@
+    let data = res->LogicUtils.getDictFromJsonObject->LogicUtils.getArrayFromDict("hits", [])
+    let total = res->getDictFromJsonObject->getInt("count", 0)
+
+    setData(offset, setOffset, total, data, setTotalCount, setTableData, setScreenState)
+  } catch {
+  | Exn.Error(_) => setScreenState(_ => PageLoaderWrapper.Error("Something went wrong!"))
+  }
+}
+
+module PreviewTable = {
+  @react.component
+  let make = (~tableData) => {
+    open PaymentIntentEntity
+    let (offset, setOffset) = React.useState(_ => 0)
+    let defaultSort: Table.sortedObject = {
+      key: "",
+      order: Table.INC,
+    }
+
+    <LoadedTable
+      visibleColumns
+      title=" "
+      hideTitle=true
+      actualData={tableData}
+      entity=tableEntity
+      resultsPerPage=10
+      totalResults={tableData->Array.length}
+      offset
+      setOffset
+      defaultSort
+      currrentFetchCount={tableData->Array.length}
+      tableLocalFilter=false
+      tableheadingClass=tableBorderClass
+      tableBorderClass
+      ignoreHeaderBg=true
+      tableDataBorderClass=tableBorderClass
+      isAnalyticsModule=false
+      showResultsPerPageSelector=false
+      paginationClass="hidden"
+    />
+  }
+}
+
+@react.component
+let make = () => {
+  open APIUtils
+  open LogicUtils
+  open PaymentIntentEntity
+  let updateDetails = useUpdateMethod()
+  let (screenState, setScreenState) = React.useState(_ => PageLoaderWrapper.Loading)
+  let (data, setData) = React.useState(_ => [])
+  let (totalCount, setTotalCount) = React.useState(_ => 0)
+  let (searchText, setSearchText) = React.useState(_ => "")
+  let widthClass = "w-full"
+  let heightClass = ""
+  let defaultValue: LoadedTable.pageDetails = {offset: 0, resultsPerPage: 10}
+  let pageDetailDict = Recoil.useRecoilValueFromAtom(LoadedTable.table_pageDetails)
+  let pageDetail = pageDetailDict->Dict.get("Orders")->Option.getOr(defaultValue)
+  let (offset, setOffset) = React.useState(_ => pageDetail.offset)
+
+  React.useEffect0(() => {
+    switch GlobalSearchBarUtils.sessionStorage.getItem(. "results")->Nullable.toOption {
+    | Some(value) => {
+        let valueDict = value->JSON.parseExn->JSON.Decode.object->Option.getOr(Dict.make())
+        let searchText = valueDict->getString("searchText", "")
+        setSearchText(_ => searchText)
+      }
+    | None => ()
+    }
+
+    None
+  })
+
+  React.useEffect2(() => {
+    if searchText->String.length > 0 {
+      getData(
+        ~updateDetails,
+        ~setTableData=setData,
+        ~setScreenState,
+        ~setOffset,
+        ~setTotalCount,
+        ~offset,
+        ~query={searchText},
+      )->ignore
+    }
+
+    None
+  }, (offset, searchText))
+
+  let customTitleStyle = ""
+
+  <ErrorBoundary>
+    <div className={`flex flex-col mx-auto h-full ${widthClass} ${heightClass} min-h-[50vh]`}>
+      <PageUtils.PageHeading title="Payment Intent" customTitleStyle />
+      <PageLoaderWrapper screenState>
+        <LoadedTable
```

Comment:
From LoadedTable.res

Remove this piece of code - 

```

  React.useEffect0(_ => {
    if title->isEmptyString && GlobalVars.isLocalhost {
      showPopUp({
        popUpType: (Denied, WithIcon),
        heading: `Title cannot be empty!`,
        description: React.string(`Please put valid title and use hideTitle prop to hide the title as offset recoil uses title`),
        handleConfirm: {text: "OK"},
      })
    }
    None
  })

```
So you don't need to pass the title everytime. It is not in our use case